packages/thumbprint-react/components/BlockList/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './index.module.scss';
+
+export function BlockListItemLink({ children, to, className }) {
+    return (
+        <a
+            href={to}
+            className={classNames({
+                [styles.itemLink]: true,
+                [className]: className !== undefined,
+            })}
+        >
+            {children}
+        </a>
+    );
+}
+
+export function BlockListItem({ children }) {
+    return <div className={styles.item}>{children}</div>;
+}
+
+export function BlockList({ children, flush, border }) {
+    return (
+        <div
+            className={classNames({
+                [styles.list]: true,
+                [styles.listFlush]: flush === true,
+                [styles.listBorderGroup]: border === 'group',
+                [styles.listBorderBottom]: border === 'bottom',
+            })}
+        >
+            {children}
+        </div>
+    );
+}
+
+BlockList.propTypes = {
+    /**
+     * Add a border to the bottom or surround the entire list.
+     */
+    border: PropTypes.oneOf(['bottom', 'group']),
+    /**
+     * Remove padding on left and right side of each list item.
+     */
+    flush: PropTypes.bool,
```

Comment:
We've never used `tp-block-list--flush`. Should we leave it out until we have a use case?

Reply:
Sure.